### DISCUSSION TITLE
Explore options on how to move forward with the TAG ENV landscape document

### DISCUSSION THREAD

**Discussion by leonardpahlke** [cncf/tag-env-sustainability#276] [General]
Hey all, the TAG ENV worked on a [landscape document](https://tag-env-sustainability.cncf.io/landscape/). The landscape was published as an early preview, with gaps we knew about. After we published the document (early 2023) not a lot changed. The landscape document is an important artifact which can help the community. We should find a way how we can move forward. 

1. Which purpose serves the landscape in the community, and does the current form solve this purpose best?
2. How can we drive the landscape document as a project to make sure it has some velocity (perhaps velocity will be achived if we get the purpose question right)
3. Who takes ownership

cc @mkorbi @catblade

**Comment by all-green-dev** [comment]
To the 1. question: I believe the landscape document should:

1. provide an overview over the current state of sustainable cloud tech. This includes:
     a.     an introduction of central ideas, concepts, and practices,
     b.     anything that goes beyond an introduction should be moved into a separate document, which is then referenced by the landscape document, for further reading,
     c.     Maybe it could also highlight not only the achievements and best-practices, which we have and know, but also where we are currently lacking and what we are researching/working on, plus links to how and where to contribute, 
2. be targeted towards readers with prior knowledge about cloud technology. It should specifically target:
     a.     Data Center Operators,
     b.     Cloud Service Providers,
     c.     Cloud and Software Engineers,
     d.     Perhaps End Users too.
 
To the 2. and 3. question: I have some ideas on how to restructure, refocuse and ultimately rewrite the landscape document.
I propose the following structure (This should only be an overview of the rough layout and not the actual headlines):

1. Introduction
2. General Concepts and Fundamentals
4. Shared responsibility model
5. Topics for Cloud Services Providers & Data Center Operators
6. Topics for (Cloud) Software Engineers
7. Aspects for End Users 
8. Further Reading

I’d be happy to write something up as a first draft for a landscape version V1.0, if you are interested.
I know that this is no long-term solution to provide velocity, but perhaps it can kick-start it? I believe that for the future the landscape doc must find a home at a TAG WG.


Note: As one of the co-authors of the original landscape document, I may be biased. So let me know what you think.

**Reply by catblade** [reply]
Would add also an appendix (section 8) for the available tooling so we don't have to repeat content.  Mostly in agreement otherwise.

**Comment by leonardpahlke** [comment]
Further collab: https://github.com/cncf/tag-env-sustainability/issues/302
